HyperIndex_versioned_docs/version-v1/contract-import.md

```diff
@@ -1,7 +1,7 @@
 ---
 id: contract-import
-title: No-Code Quickstart
-sidebar_label: No-Code Quickstart
+title: Quickstart
+sidebar_label: Quickstart
```

Comment:
@SvenMuller95 we need to make sure in the remappings that this is correctly redirected too

Reply:
Hey @moose-code , I don't think this affects the slug, URL or need redirect. Neither the sidebar.js  It's merely title and sidebar label (as it appears in the docs).